.github/workflows/debos.yml

```diff
@@ -35,6 +35,12 @@ jobs:
         - /efs/qli/metaqcom/gh-runners/quic-yocto/downloads:/fileserver-downloads
       options: --privileged
     steps:
+      - name: TMP-DEBUG
+        uses: qualcomm-linux/upload-private-artifact-action@v1
+        with:
+          path: /tmp/${BUILD_ID}
+          build_id: "${{ github.repository }}-${{ github.run_id }}-${{ github.run_attempt }}"
```

Comment:
This is more some thoughts on the API of this action, but I wonder if we should keep the concept of a build_id local to the repositories that needs one.

As an user of such an action, I would expect it to provide similar behavior as actions/upload-artifact. In particular, I expect that the artifacts uploaded from one repo are stored in a repo specific collection of artifacts.

When using upload-artifact, most parameters are optional. Perhaps our upload-private-artifact could provide similar behavior and there would be a download action to easily retrieve a particular artifact or the latest artifact with some name (like we're doing in the debos recipe to fetch u-boot/linux builds prior to building an image).